packages/language-server/src/server.ts

```diff
@@ -46,41 +41,32 @@ function getConnection(options?: LSOptions): IConnection {
   return connection
 }
 
+let hasCodeActionLiteralsCapability = false
+let hasConfigurationCapability = false
+let defaultBinPath = ''
```

Comment:
Does this need to be a global variable? I think it is only used in the `startServer` function. Maybe we can move it inside and make it something like 

`const defaultBinPath = await util.getBinPath()`

Reply:
I can move it inside the `startServer` function, but I cannot make it `const`, otherwise the `startServer` function would need to be made async.

Reply:
Ah, I got it, it gets initialized in an async callback. All good, the existing setup is also fine then 👍